src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java

```diff
@@ -330,6 +330,21 @@ private void setAfterSubmitCallbackResponseEntity(final AfterSubmitCallbackRespo
         return metadata;
     }
 
+    @JsonIgnore
+    public Map<String, Object> getFormattedMetadata() {
+        if (metadata.isEmpty()) {
+            metadata.put(JURISDICTION.getReference(), getJurisdiction());
+            metadata.put(CASE_TYPE.getReference(), getCaseTypeId());
+            metadata.put(STATE.getReference(), getState());
+            metadata.put(CASE_REFERENCE.getReference(), getReference() != null ? getReference().toString() : getId());
```

Comment:
for my understanding, can you explain where the code breaks if you don't do this conversion? maybe we can find a better way to do this

Reply:
Toolkit SearchResultViewItemComparatorFactory comparators. Browser errors